dsn_test.go

```diff
@@ -339,6 +339,22 @@ func TestParseDSN(t *testing.T) {
 			},
 			ocspMode: ocspModeFailOpen,
 		},
+		{
+			dsn: "u:p@a?database=d&maxRetryCount=20",
```

Comment:
Can we verify that the default value is correctly set, if not configured?

Reply:
Added a check in other test.